### PR TITLE
Removed the requirement that the buttons are actually buttons, they c…

### PR DESCRIPTION
**Pull request by btkelly**
…an now be just views

### CODE REVIEW

File: amplify/src/main/java/com/github/stkent/amplify/views/DefaultQuestionView.java [hunk 1]
```diff
@@ -44,12 +45,12 @@ public DefaultQuestionView(@NonNull final View view) {
             throw new IllegalArgumentException("You must supply a layout that includes a TextView with id amp_title_text_view");
         }
 
-        positiveButton = (Button) view.findViewById(R.id.amp_positive_button);
+        positiveButton = view.findViewById(R.id.amp_positive_button);
         if (positiveButton == null) {
             throw new IllegalArgumentException("You must supply a layout that includes a Button with id amp_positive_button");
```
Comment (stkent) [inline amplify/src/main/java/com/github/stkent/amplify/views/DefaultQuestionView.java]: This string needs updating.

File: amplify/src/main/java/com/github/stkent/amplify/views/DefaultQuestionView.java [hunk 2]
```diff
@@ -44,12 +45,12 @@ public DefaultQuestionView(@NonNull final View view) {
...
         }
 
-        negativeButton = (Button) view.findViewById(R.id.amp_negative_button);
+        negativeButton = view.findViewById(R.id.amp_negative_button);
         if (negativeButton == null) {
             throw new IllegalArgumentException("You must supply a layout that includes a Button with id amp_negative_button");
```
Comment (stkent) [inline amplify/src/main/java/com/github/stkent/amplify/views/DefaultQuestionView.java]: This string needs updating.

File: amplify/src/main/java/com/github/stkent/amplify/views/QuestionView.java [hunk 3]
```diff
@@ -26,11 +25,11 @@
...
     @NonNull
-    Button getPositiveButton();
+    View getPositiveButton();
 
     @NonNull
-    Button getNegativeButton();
+    View getNegativeButton();
 
-    void setQuestion(@NonNull final Question question);
+    void setQuestion(@NonNull final String question);
```
Comment (stkent) [inline amplify/src/main/java/com/github/stkent/amplify/views/QuestionView.java]: This should be renamed `setTitle` now.

Comment (btkelly) [inline amplify/src/main/java/com/github/stkent/amplify/views/QuestionView.java]: Hmm not sure, it's still just the question string. Right now there isn't a title, maybe we will add it tomorrow? We can hold off on this PR till we can look at this in person.

**Comment by stkent** [comment]
LGTM on success!

**Pull request merged**